# teleoperation config
default_hostname: "127.0.0.1"
default_robot_port: 6001
control_hz: 100
agent_name: "gello" # "gello" or "act"

# camera config
default_camera_port: 7001  # If there are multiple cameras, use consecutive numbers
camera_names: ["base"]
camera_width: 640
camera_height: 480

# imitation learning config
task_name: "test_move"
save_episode_dir: '/root/onolab/catkin_ws/src/gello_software/gello_ros/scripts/episode_data'  # Directory for saving datasets
number_of_episodes: 10
number_of_steps: 100
state_dim: 6
action_dim: 6


